dags/bqetl_activity_stream.py

```diff
@@ -36,6 +36,7 @@
         task_id="wait_for_copy_deduplicate_copy_deduplicate_all",
         external_dag_id="copy_deduplicate",
         external_task_id="copy_deduplicate_all",
+        execution_delta=datetime.timedelta(days=-1, seconds=82800),
```

Comment:
This is equivalent to `timedelta(hours=-1)`

Reply:
This is strange enough that I think it's worth fixing as part of this PR.

Reply:
Fix it as in replace it with `timedelta(hours=-1)`?

Because in the python shell I also get:

```
>>> timedelta(hours=-1)
datetime.timedelta(-1, 82800)
```

Reply:
> Fix it as in replace it with `timedelta(hours=-1)`?

Yes, that's what I meant. But you make a good point that `datetime.timedelta(-1, 82800)` is the canonical form. I'm okay with moving forward, but I will file an issue for this, because I think it's going to be confusing for anyone looking at the text of the DAG.

Reply:
`execution_delta` always confuses me. Looking at it more closely, it needs to be timedelta(hours=1) for the DAGs here.